modules/network-observability-configuring-netobserv-dashboard.adoc

```diff
@@ -0,0 +1,26 @@
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard_{context}"]
+= Configuring viewable metrics for the Netobserv dashboard
+You can add or remove metrics from the *Netobserv* dashboard by editing the `ignoreTags` list in the `FlowCollector` `spec.processor.metrics` parameter. 
+
+.Procedure
+. In the web console, navigate to Operators → Installed Operators.
+. Under the Provided APIs heading for the NetObserv Operator, select Flow Collector.
```

Comment:
```suggestion
. Under the Provided APIs heading for Network observability, select Flow Collector.
```

Reply:
I don't have access to a cluster atm, but did we actually change this to Network Observability in the console? It used to be NetObserv.

Reply:
In most places we renamed NetObserv to Network Observability, but this isn't new, I think it dates from our first GA :-) 
I know there's a couple of places where this isn't the case, especially for texts in code, but here this name is taken from the operator metadata which we renamed.

Here's a screenshot:
![image](https://github.com/openshift/openshift-docs/assets/2153442/eea29a9b-6d16-4139-a748-f0c2e9083484)